INSTALL.md

```diff
@@ -1,81 +1,55 @@
+2. Connect your device to the computer via USB.
 
+## 1. Device preparation
 
-## Device preparation
-
-##### Enable developer mode
-1. Go to **Settings > About tablet**.
+##### Enable developer mode on your device
+1. Go to **Settings --> About Phone/tablet**.
 2. Tap **Build Number** 7 times until it displays _"You are now a developer"_.
 
-
-##### Enable OEM Unlocking
-1. Go to **Settings > Developer** options.
-2. Enable OEM Unlocking.
-3. Tap **Enable** when prompted by the disclaimer.
-
-##### Enable USB Debugging
-1. Go to **Settings > Developer** options.
+##### Enable USB Debugging on your device
+1. Go to **Settings --> Developer** options.
 2. Tap **Enable USB debugging**.
 3. Tap **OK** when prompted by the disclaimer.
-4. You should see dialog with you computer's _"RSA key fingerprint"_.
+4. You should see a dialog box with your computer's _"RSA key fingerprint"_.
 5. Tap _"Always allow from this computer"_.
 6. Tap **OK**.
 
-## Unlock the bootloader
-1. On you computer, reboot the device into the bootloader by running:
-
-        adb reboot-bootloader
- 
-2. The device will now boot into the bootloader and display _Waiting for fastboot command…_.
-
-3. Unlock the bootloader by running:
- 
-        fastboot flashing unlock
-
-4. The device will display a warning about unlocking the bootloader.
-5. Press the **power** button on the device to unlock the bootloader.
-6. The device will reboot into the bootloader and display _Waiting for fastboot command…_.
-7. Reboot the device back into Android by running:
- 
-        fastboot reboot
-
-8. Since your device will be reset, you will need to [enable USB debugging](#enable-usb-debugging) once more.
-
+## 2. Building the Pehla Class app
 
+1. Ensure you have **Android Studio** installed. To do this, you can [follow the steps in this helpful guide](https://developer.android.com/studio/install.html).
+     
+2. Start Android Studio and import the project into it.
 
-## Flash filesystem images
+3. In order to build the app for a specific language, select the desired language from the build variant.
 
-1. Download the set of filesystem images for the version of onecourse you wish to install:
- - onecourse Swahili [part 1](https://github.com/XPRIZE/GLEXP-Team-onebillion/releases/download/v3.0.1/onecourse-swahili-v3.0.1.tar.gz.aa), [part 2](https://github.com/XPRIZE/GLEXP-Team-onebillion/releases/download/v3.0.1/onecourse-swahili-v3.0.1.tar.gz.ab)
- - onecourse English [part 1](https://github.com/XPRIZE/GLEXP-Team-onebillion/releases/download/v3.0.1/onecourse-english-v3.0.1.tar.gz.aa), [part 2](https://github.com/XPRIZE/GLEXP-Team-onebillion/releases/download/v3.0.1/onecourse-english-v3.0.1.tar.gz.ab)
+4. Add google-services.json file in language specific [app](https://github.com/maqsoftware/Pehla-Class/tree/master/app/src) folder. The google-service.json file is available in the Firebase console after a new Android app has been added to a Firebase project.
 
```

Comment:
`google-services.json`. Also, the s is missing in one of its name instance.
Update the line to this "..  after registering a new app in Firebase console"